pkg/containerruntime/docker_config.go

```diff
@@ -33,8 +33,8 @@ type dockerConfig struct {
 	RegistryMirrors    []string          `json:"registry-mirrors,omitempty"`
 }
 
-func marshalDockerConfig(cluster *kubeone.KubeOneCluster) (string, error) {
-	// Parse log max size to ensure that it has the correct units
+// Parse log max size to ensure that it has the correct units
```

Comment:
This comment is used to describe the `logSize` logic, not the function.

Reply:
yeah, you're right, incorrect rebase